spanner/spannertest/db.go

```diff
@@ -622,6 +622,10 @@ func (t *table) addColumn(cd spansql.ColumnDef, newTable bool) *status.Status {
 		return status.Newf(codes.InvalidArgument, "new non-key columns cannot be NOT NULL")
 	}
 
+	if _, ok := t.colIndex[cd.Name]; ok {
+		return status.Newf(codes.InvalidArgument, "duplicated column names: %s", cd.Name)
```

Comment:
Violation of the uniqueness of a column name for a table should use error code `AlreadyExists`:

```suggestion
		return status.Newf(codes.AlreadyExists, "column %s already exists", cd.Name)
```

Reply:
I see. Thanks, fixed now.